### DISCUSSION TITLE
tenacityaudio.org/ download redirect to release page of github

### DISCUSSION THREAD

**Discussion by softyoda** [tenacityteam/tenacity-legacy#736] [Q&A] [answered]
When I click on download on the page https://tenacityaudio.org/ it redirect me to https://github.com/tenacityteam/tenacity/releases 

### Expected behavior

It should redirect me to https://nightly.link/tenacityteam/tenacity/workflows/cmake_build/master

**Comment by softyoda** [comment]
Also, is there any download for windows 10 pro (I don't have windows server installed on any computer at home)

**Comment by fitojb** [comment, marked as answer]
This project has never officially released stable binaries, hence the lack of a download. Also, Windows Server and Windows Pro are just two different configurations of the same operating system. The link will work just fine in your system.
